runtime/shared_heap.c

```diff
@@ -34,7 +34,11 @@
 #include "caml/startup_aux.h"
 
 typedef unsigned int sizeclass;
-struct global_heap_state caml_global_heap_state = {0 << 8, 1 << 8, 2 << 8};
+struct global_heap_state caml_global_heap_state = {
+        0 << HEADER_COLOR_SHIFT, /* MARKED */
+        1 << HEADER_COLOR_SHIFT, /* UNMARKED */
+        2 << HEADER_COLOR_SHIFT, /* GARBAGE */
```

Comment:
```suggestion
        0 << HEADER_COLOR_SHIFT,
        1 << HEADER_COLOR_SHIFT,
        2 << HEADER_COLOR_SHIFT,
```
I don't think the comments are useful, since any permutation of these lines would work just as well.